.github/workflows/build-test-format-linux.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
```

Comment:
don't change this file, we are going to include a new workflow for testting the apps ina  separate PR